File: exo6/exo6.py
#perroquet
#author sebastien
#beWeb

quitter = False
reponse = ""
flag = False

while quitter == False:
	perroquet = raw_input("entrez une phrase ou un mot:\r\n")
	print(str(perroquet))
	flag = True

	while flag == True:
		reponse = raw_input("Voulez vous quitter o/n ?\r\n")

		if "o" == reponse:
			quitter = True
			flag = False
		elif "n" == reponse:
			flag = False
		else:
			print("J'ai pas compris")

File: exo6/exo6.php
<?php
//perroquet
//author sebastien
//beWeb

$quitter = false;
$reponse = "";
$flag = false;

while ($quitter == false) {
	echo("Entrer une phrase ou un mot:\r\n");
	$reponse = fgets(STDOUT);
	echo $reponse;
	$flag = true;

	while ($flag == true){
		echo("Voulez vous quitter o/n ?\r\n");
		$reponse = fgets(STDOUT);
		if ($reponse == "o\n") {
			$quitter = true;
			$flag = false;
		}elseif ($reponse ==  "n\n"){
			$flag = false;
		}else{
			echo("J'ai pas compris\r\n");
		}
	}
}
?>
